scipy/special/_precompute/expn_asy.py

```diff
@@ -10,16 +10,11 @@
 from __future__ import division, print_function, absolute_import
 
 import os
-import warnings
 
 try:
-    # Can remove when sympy #11255 is resolved; see
-    # https://github.com/sympy/sympy/issues/11255
-    with warnings.catch_warnings():
```

Comment:
@person142 Oops, makes sense. Amended to add `suppress_warnings()` there (and in `utils.py`).